### DISCUSSION TITLE
Is Nuxt Bridge Ready for Production?

### DISCUSSION THREAD

**Discussion by vencho-mdp** [nuxt/nuxt#17715] [Questions] [answered]
If we start a prod project right now, should we use: Nuxt 2, Bridge  or Nuxt 3?

BTW, as a developer I'd love to use Nuxt 3 because of its amazing features, but stability is quite important in a production level project.

**Comment by marcusaram** [comment, marked as answer]
If it is anything worth. We run a pure Nuxt 3 project in production (without bridge) without issues. This is a small project with a few pages so we took the risk. We don't have any issues so far.
